apps/academy-wep/components/FeaturedLearningPaths.tsx

```
@@ -0,0 +1,63 @@
+        <span className="font-bold text-green-700"> Journeys</span>
+      </h1>
+      <p className="text-gray-600 mb-6 text-lg md:text-xl">
+        Follow our curated learning paths to master blockchain technologies in a
+        structured, step-by-step approach.
+      </p>
+
+      <div className="flex flex-col md:flex-row md:space-x-6 space-y-6 md:space-y-0 p-2">
+        {/* Left Section: Learning Path Cards */}
+        <div className="flex flex-col space-y-4 md:flex-row md:space-y-0 md:space-x-6 md:w-full">
+          <LearningPathCard
+            icon={<i className="fas fa-shield-alt"></i>}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)